sgpt/handlers/repl_handler.py

```diff
@@ -14,6 +14,15 @@ class ReplHandler(ChatHandler):
     def __init__(self, chat_id: str, role: SystemRole) -> None:
         super().__init__(chat_id, role)
 
+    def get_multiline_input(self) -> str:
```

Comment:
This could be shorter using walrus operator, but not an issue.
```python
def get_multiline_input(self) -> str:
    multiline_input = ""
    while (user_input := typer.prompt("...", prompt_suffix="")) != '"""':
        multiline_input += user_input + "\n"
    return multiline_input
```